src/test/java/org/jenkins_ci/plugins/flexible_publish/ConfigurationTest.java

```diff
@@ -430,44 +467,8 @@ public void testMatrixDisableAggregationCondition() throws Exception {
         assertEquals("anotherProject", trigger.getChildProjectsValue());
         assertEquals(Result.SUCCESS, trigger.getThreshold());
     }
-    
-    public void testNoDataBoundConstructor() throws Exception {
```

Comment:
Note for me: Create an alternate test: https://issues.jenkins.io/browse/JENKINS-64527